c2corg_api/models/report.py

```diff
@@ -0,0 +1,298 @@
+import colander
+from c2corg_api.models.schema_utils import restrict_schema,\
+    get_update_schema, get_create_schema
+from c2corg_common.fields_report import fields_report
+from sqlalchemy import (
+    Column,
+    Integer,
+    Boolean,
+    SmallInteger,
+    ForeignKey,
+    Float,
+    String,
+    Date
+    )
+
+from colanderalchemy import SQLAlchemySchemaNode
+
+from c2corg_api.models import schema, Base
+from c2corg_api.models.utils import ArrayOfEnum
+from c2corg_api.models.utils import copy_attributes
+from c2corg_api.models.document import (
+    ArchiveDocument,
+    Document,
+    geometry_schema_overrides,
+    schema_attributes, DocumentLocale, ArchiveDocumentLocale,
+    schema_locale_attributes)
+from c2corg_common import document_types
+from c2corg_api.models import enums
+
+REPORT_TYPE = document_types.REPORT_TYPE
+
+
+class _ReportMixin(object):
+    # Event characterization-Caractérisation de l'évènement
+
+    # TODO DECIDE IF IT SHOUD BE HERE OR NO
```

Comment:
Lon/lat are stored as geometry. Only in the UI, you see lon/lat as explicit fields. Same as for example for waypoints.